lib/cassia/response_handlers/get_all_routers_status.rb

```diff
@@ -0,0 +1,53 @@
+module Cassia
+  module ResponseHandlers
+    class GetAllRoutersStatus
+      def initialize(access_controller)
+        @access_controller = access_controller
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success(response)
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success(response)
+        response.body.each do |router|
+          new_router = Router.new
```

Comment:
I think if you named the variables the same as on the response you can do something like this 
```
new_router = Router.new(router)
@access_controller.routers << new_router
```

I think it can match attributes based on the hash you pass in, but I could be wrong